data/migrations/V0087__operations_log_mv.sql

```diff
@@ -0,0 +1,48 @@
+-- Materialized View: public.ofec_operations_log_mv
+
+DROP MATERIALIZED VIEW IF EXISTS public.ofec_operations_log_mv;
+
+SET search_path = public;
+
+CREATE MATERIALIZED VIEW public.ofec_operations_log_mv AS 
+ SELECT o.sub_id,
+    o.status_num,
+    o.cand_cmte_id,
+    o.beg_image_num,
+    o.end_image_num,
+        CASE
```

Comment:
These 4 lines can be replaced with `COALESCE(f.form_tp, regexp_replace(o.form_tp, 'A|N|T', ''))`